### PR TITLE
Prep for 0.9.1 release

### PR DESCRIPTION
**Pull request by swenson**
* Update deps
* Update go
* Update README
* Update changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
 
+## v0.9.1
+
+IMPROVEMENTS:
+* Updated dependencies:
+   * `github.com/couchbase/gocb/v2` v2.3.3 -> v2.6.3
+   * `github.com/hashicorp/go-hclog` v1.0.0 -> v1.5.0
+   * `github.com/hashicorp/go-version` v1.3.0 -> v1.6.0
+   * `github.com/hashicorp/vault/sdk` v0.5.3 -> v0.9.0
+   * `github.com/mitchellh/mapstructure` v1.5.0 -> v1.5.0
```
Comment (tomhjp) [inline CHANGELOG.md:11]: Looks like there's a small bug in the python script for deps that moved from indirect to direct?

File: README.md [hunk 2]
```diff
@@ -142,3 +140,9 @@ username               vault-edu
 ## Developing
 
 You can run `make dev-vault` in the root of the repo to start up a development vault server and automatically register a local build of the plugin. You will need to have a built `vault` binary available in your `$PATH` to do so.
+
+### Acceptance tests
+
+Run `make testacc`.
+
+For ARM-based macs, you need to run with `make testacc COUCHBASE_VERSION=7.1.3`.
```
Comment (tomhjp) [inline README.md:148]: Could maybe update the tests to default to couchbase 7? It seems unlikely there will be breakages in 6 anyway now that 7 is the latest major.

**Review by tomhjp — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (swenson) [reply CHANGELOG.md:11]: Ah, good catch. I'll fix that. Thanks.

**Comment by swenson** [comment]
@tomhjp cleaned up version logic a little in the tests. PTAL

File: couchbase_test.go [hunk 3]
```diff
@@ -16,10 +16,12 @@ import (
 	dbtesting "github.com/hashicorp/vault/sdk/database/dbplugin/v5/testing"
 	"github.com/ory/dockertest/v3"
 	dc "github.com/ory/dockertest/v3/docker"
+	"github.com/rogpeppe/go-internal/semver"
```
Comment (tomhjp) [inline couchbase_test.go:19]: Maybe use [hashicorp/go-version](https://github.com/hashicorp/go-version) instead?

**Review by tomhjp — APPROVED** [review]

File: couchbase_test.go (hunk 3 above)
Comment (swenson) [reply couchbase_test.go:19]: Oh, I had meant to use the golang.org/x one. Not sure why this got pulled in. Thanks for the catch.

**Comment by swenson** [comment]
Thanks!

**Pull request merged**